changed.eslintrc.js

```diff
@@ -1,21 +0,0 @@
-const eslintConfig = require('./.eslintrc');
-
-// Add the config for preventing deprecated component-library imports
-eslintConfig.plugins = [...eslintConfig.plugins, 'deprecate'];
-eslintConfig.rules = {
-  ...eslintConfig.rules,
-  'deprecate/import': [
-    'error',
```

Comment:
This rule is moved into the main ESLint config with the exception that `error` is changed to `warn`.